pkg/resource/conditions.go

```diff
@@ -26,11 +26,14 @@ import (
 
 // Condition constants.
 const (
-	TypeAsyncOperation = "AsyncOperation"
+	TypeAsyncOperation         = "AsyncOperation"
+	TypeAsyncOperationFinished = "AsyncOperationFinished"
```

Comment:
Maybe `AsyncOperation` for the ongoing one and `LastAsyncOperation` for the result of the finished one?